src/System.Management.Automation/engine/interpreter/CallInstruction.Generated.cs

```diff
@@ -130,7 +130,7 @@ internal partial class CallInstruction {
                 case TypeCode.UInt16: return FastCreate<UInt16>(target, pi);
                 case TypeCode.UInt32: return FastCreate<UInt32>(target, pi);
                 case TypeCode.UInt64: return FastCreate<UInt64>(target, pi);
-                case TypeCode.String: return FastCreate<String>(target, pi);
+                case TypeCode.String: return FastCreate<string>(target, pi);
```

Comment:
This shouldn't be changed. We are dealing with primitive types in general here.

Reply:
```suggestion
                case TypeCode.String: return FastCreate<String>(target, pi);
```

Reply:
Fixed.